modules/theme-tools/devicepx.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Loads the devicepx library which improves the resolution of gravatars and
+ * wordpress.com uploads on hi-res and zoomed browsers.
+ *
+ * This feature will only be activated for themes that declare their support.
+ * This can be done by adding code similar to the following during the
+ * 'after_setup_theme' action:
+ *
+ * add_theme_support( 'jetpack-devicepx' );
+ */
+
+/**
+ * Add hooks to enqueue the devicepx JS library, if enabled. The feature must
+ * be enabled earlier during `after_setup_theme`.
+ *
+ * @uses current_theme_supports, add_action, Jetpack::is_active
+ */
+function jetpack_devicepx_init() {
+	if ( Jetpack::is_active() && current_theme_supports( 'jetpack-devicepx' ) ) {
+		add_action( 'wp_enqueue_scripts', 'jetpack_devicepx_enqueue' );
+		add_action( 'customize_controls_enqueue_scripts', 'jetpack_devicepx_enqueue' );
+		add_action( 'admin_enqueue_scripts', 'jetpack_devicepx_enqueue' );
+	}
+}
+add_action( 'init', 'jetpack_devicepx_init', 99 );
```

Comment:
This must fire after priority 1 which is used by the AMP module to `remove_theme_support( 'jetpack-devicepx' )`.

Reply:
Do you think it is worth noting that in an inline comment?